app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class SessionsController < ApplicationController
+  def new
+    if logged_in?
+      redirect_to root_url
+    else
+      render 'new'
+    end
+  end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user
+      log_in user
+      remember user
+      flash[:succsess] = 'Welcome'
+      redirect_to root_url
+    else
+      flash[:errors] = ['Invalid e-mail']
+      redirect_back(fallback_location: root_path)
+    end
+  end
+
+  def destroy
+    user = current_user
+    sign_out user
+    forget user
```

Comment:
Don't forget to also update the remember digest attribute in the database for the user that is signing out. This needs to be updated to nil after sign out. This could be achieved with a forget method in the User model that updates the remember digest attribute to nil. [OPTIONAL]